.github/pull_request_template.md

```diff
@@ -1,13 +1,20 @@
+<!-- Thank you for offering your contribution with Deals for Devs! -->
+
 ## Description
-<!-- Describe your contribution and its benefits in a sentence  -->
+<!-- Briefly explain what this PRs purpose is. -->
```

Comment:
[suggestion] It'd be best to describe them as proposed changes. Some PRs get rejected.  
```suggestion
<!-- Briefly describe your PR‘s proposed changes. -->
```